Makefile

```diff
@@ -0,0 +1,83 @@
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# This is a requirement for 'setup-envtest.sh' in the test target.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
+
+.PHONY: all
+all: lint
+
+##@ General
+
+# The help target prints out all targets with their descriptions organized
+# beneath their categories. The categories are represented by '##@' and the
+# target descriptions by '##'. The awk commands is responsible for reading the
+# entire set of makefiles included in this invocation, looking for lines of the
+# file as xyz: ## something, and then pretty-format the target and help. Then,
+# if there's a line with ##@ something, that gets pretty-printed as a category.
+# More info on the usage of ANSI control characters for terminal formatting:
+# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
+# More info on the awk command:
+# http://linuxcommand.org/lc3_adv_awk.php
+
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Development
+
+.PHONY: lint
+lint:  lint-cache ## Run protoc-gen-lint against the whole API
+
+.PHONY: lint-cache
+lint-cache: protoc protoc-gen-lint ## Run protoc-gen-lint against protobuf API
+	PATH=$(LOCALBIN):$(PATH) $(PROTOC) 	--lint_out=. --lint_out=sort_imports:. \
+	          	--lint_opt=Mconfig/cache/v1alpha1/cache.proto=github.com/gingersnap-project/operator/api/v1alpha1,Mconfig/cache/v1alpha1/rules.proto=github.com/gingersnap-project/operator/api/v1alpha1 \
+			  	config/cache/v1alpha1/*.proto
+
+# Add here lint command for grpc services
+# .PHONY: lint-service
+# lint-service: protoc protoc-gen-lint ## Run protoc-gen-lint against grpc API
+
+
+
+##@ Build Dependencies
```

Comment:
Outdated comments and whitespace can be removed.

Reply:
Removed unneeded comments.
I think `Build Dependencies` is ok, it groups targets the same way as the operator in `make help`